xcode/Subconscious/Shared/Components/Detail/DetailStackView.swift

```diff
@@ -282,8 +291,8 @@ struct DetailStackModel: Hashable, ModelProtocol {
         model.details = details
         return Update(state: model)
     }
-
-    static func findAndPushDetail(
+    
+    static func findAndPushLinkDetail(
```

Comment:
Introduced this new intermediate action for links which will attempt to find the shortest peer path we can. This makes it simpler to support tappable links in the feed.